packages/identicon/src/IdenticonStyles.tsx

```diff
@@ -15,21 +15,13 @@ const colors = {
 export const IdenticonBadgeStyle = styled.div<{ $color: IdenticonBadgeColors }>`
   border-radius: 1000px;
   padding: 4px;
+  margin: 0 6px;
```

Comment:
This margin applies to the outer element, not the inner one.

The result:
![image](https://user-images.githubusercontent.com/1422379/128596534-a86f0d10-cde5-4492-848b-7e00e9d32c6c.png)

Before changes:
![image](https://user-images.githubusercontent.com/1422379/128596670-df99cfa1-90cf-42cc-a574-5014e7dc2fc2.png)

Reply:
fixed

Reply:
Great, thanks! Could you add the same padding on the left side? It still needs fixing:

![image](https://user-images.githubusercontent.com/1422379/130487699-6ea294be-1635-4f75-b1d9-ba52569808a5.png)

Reply:
fixed

Reply:
fixed for empty badge (without address)